stable/elasticsearch/values.yaml

```diff
@@ -21,6 +21,15 @@ serviceAccounts:
 ##
 podSecurityPolicy:
   enabled: false
+  annotations: {}
```

Comment:
Should we set some defaults for this?
Does it need any defaults or are the base settings ok?

Reply:
I think we should leave this configuration to the user, it's difficult to set some defaults as each k8s environment can be very different from one to an other.

Reply:
That would leave these unconfigured.
Could we set this to the defaults from above?

Reply:
If they are different for each security provider we should put a note to that effect.